swagger.yaml

```diff
@@ -1302,6 +1302,7 @@ definitions:
         - edge
         - passthrough
         - reencrypt
+        - default
```

Comment:
Will merge. However, I'm wondering how we should document these "outliers" that may not have much of a meaning at first glance. Would not want to have to dig through implementation code to find out, so we need to come up with something. I think maybe using comments/description more in this definition would be helpful.